coordinator/core/core_test.go

```diff
@@ -128,7 +128,7 @@ func TestRecover(t *testing.T) {
 	assert.Error(err)
 
 	// Set manifest. This will seal the state.
-	_, err = c.SetManifest(context.TODO(), []byte(test.ManifestJSON))
+	_, err = c.SetManifest(context.TODO(), []byte(test.ManifestJSONWithRecoveryKey))
```

Comment:
I think the original intention was to test if user permissions persisted after sealing, but since this should be done in different places, this can be removed